controller/testingClass.py

```diff
@@ -0,0 +1,5 @@
+class TestClass:
```

Comment:
I'd want to rename this file and the class to be more specific to what you're testing.